src/Network/AWS/Flow.hs

```diff
@@ -36,6 +37,7 @@ import           Safe
 
 register :: MonadFlow m => Plan -> m ()
 register Plan{..} = do
+  logInfoN "event=register\n"
```

Comment:
Yeah, I'm trying to pick a better formatting library :( Probably going to stick with formatting, as I couldn't find anything reasonable to use `text-format`. So, updated log lines are coming.